playground/keycloak/Keycloak.AppHost/Program.cs

```diff
@@ -0,0 +1,15 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+var keycloak = builder.AddKeycloak("keycloak", 8080)
```

Comment:
1. If you commonly get an error from not specifying a stable port, should we document the recommendation?
2. Is this port also used when the app is deployed? Is that the correct thing to do?

Reply:
Added a comment to the README's on why you should use a stable port during local dev.

For Production, the port won't be changing (in fact, you don't usually use ports in PROD), so this will not be an issue.